kairon/api/app/routers/bot/data.py

```
@@ -170,6 +171,30 @@
     }
 
 
+@router.delete("/cognition", response_model=Response)
+async def delete_multiple_cognition_data(
+        row_ids: list[str] = Query(...),
```

Ruff (0.8.2)

Do not perform function call `Query` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)